components/organisms/Dashboard/dashboard.tsx

```diff
@@ -15,13 +15,13 @@ interface DashboardProps {
 }
 
 const Dashboard = ({ repositories }: DashboardProps): JSX.Element => {
-  const { data: insightsData, isLoading } = useInsights(repositories);
+  const { data: insightsData, isLoading } = usePullRequestsHistogram(repositories);
   const { data: contributorData, meta: contributorMeta } = useContributors(undefined, repositories);
   const router = useRouter();
   const { range = 30 } = router.query;
 
-  const compare1 = getInsights(insightsData, 30);
-  const compare2 = getInsights(insightsData, 60);
+  const compare1 = getInsights(insightsData, 0);
```

Comment:
Does the interval argument, `0`, mean get today's insights and `1`, mean yesterdays?

Reply:
Yes, the first item in the array is today, and second is yesterday. The interval used to be returned as part of the data. I'll update the naming